packages/sdk/package.json

```diff
@@ -7,15 +7,15 @@
     ".": {
       "import": "./dist/index.js"
     },
-    "./browser-wallets": {
-      "import": "./dist/browser-wallets.js"
+    "./browser-wallets/unisat": {
+      "import": "./dist/browser-wallets-unisat.js"
```

Comment:
We are using explicit exports for browser-wallets to leverage on the benefits of tree shaking. Exporting a single object will cause all its dependencies to be imported, e.g.

foo.js
```
function a() {}
function b() {}

const test = { a, b };
export { test };
```

bar.js
```
import { test } from 'foo';
test.a()
```

`test.b()` is not considered dead code by the bundler because it may be used.

Reply:
Although I favoured the initial restructure approach (which was `import { unisat } from "@ordzaar/browser-wallets"` because you can define an interface for unisat and avoid duplicating code documentation for all wallets), which favours DX here, I think it's more important to ensure that no unnecessary imports are added when consumers import our library.